tests/test-cases/Group23-VIC-Machine-Service/23-02-VCH-List.robot

```diff
@@ -111,3 +111,17 @@ Get VCH List Within Invalid Datacenter and Compute Resource
 
     Verify Return Code
     Verify Status Not Found
+
+    [Teardown]   Run Secret VIC Machine Delete Command   %{VCH-NAME}
+
+Empty VCH list returned
```

Comment:
Just to be consistent in naming with other tests: is it better if it's named something along the lines of `Get Empty VCH List When No VCH deployed`?

Reply:
Agree, will change test name to Get Empty VCH List